CHANGES.md

```diff
@@ -79,6 +79,8 @@ To be released.
      -  Removed `ActionTypeLoaderContext` class.  Use `long` instead.
      -  Renamed `StaticActionTypeLoader` to `StaticActionLoader`.
  -  Added `IActionEvaluator.IActionLoader` property.  [[#3136]]
+ -  Changed `IActionLoader.LoadAction()` to throw `InvalidActionException`
+    instead of `ArgumentException`.  [[#3140]]
```

Comment:
```suggestion
    instead of `ArgumentException` when the action cannot be instantiated.  [[#3140]]
```
How about describe detail of the situation when this exception occurs.